react/features/invite/components/add-people-dialog/native/InviteButton.js

```diff
@@ -22,12 +21,7 @@ type Props = AbstractButtonProps & {
     /**
      * Opens the add people dialog.
      */
-    _onOpenAddPeopleDialog: Function,
-
-    /**
-     * Begins the UI procedure to share the conference/room URL.
-     */
-    _onShareRoom: Function
+    _onOpenAddPeopleDialog: Function
```

Comment:
I know you are just moving this code around, but can you please drop the mapDispatchToProps? It adds a bunch of boilerplate for just running a single line of code. we can call `this.props.dispatch()` directly on the handler.